proxy/http2/Http2ConnectionState.cc

```diff
@@ -840,7 +857,10 @@ static const http2_frame_dispatch frame_handlers[HTTP2_FRAME_TYPE_MAX] = {
 int
 Http2ConnectionState::main_event_handler(int event, void *edata)
 {
-  if (edata == fini_event) {
+  if (edata == zombie_event) {
+    // zombie session is still around. Assert
+    ink_release_assert(zombie_event == nullptr);
```

Comment:
You might want something more like `ink_release_assert(!"Zombie HTTP/2 event occurred");`